haskell-backend/src/main/java/org/kframework/backend/haskell/HaskellRewriter.java

```diff
@@ -305,6 +312,10 @@ private String saveKoreSpecToTemp(ModuleToKORE converter, Module rules) {
                     args.add("--smt-prelude");
                     args.add(smtOptions.smtPrelude);
                 }
+                if (kProveOptions.debugScript != null) {
+                    args.add("--repl-script");
+                    args.add(files.resolveWorkingDirectory(kProveOptions.debugScript).getAbsolutePath());
+                }
```

Comment:
Should we only enable this option if `--debugger` is passed? The only other use-case I can think of is if they pass `--haskell-backend-command` directly, and also want to use this option, but then tehy can just add `--repl-script` to their `--haskell-backend-command`.

Reply:
I mean, if anything it should be an error to pass this flag but not --debugger. I can implement that behavior if you'd like.

Reply:
Up to you, I don't think it's worth blocking the merge. But yes, that's what I meant, this flag should only be passed if `--debugger` is passed. If it's simple, we might as well do it I think.